### PR TITLE
Model 9 Stillbirth Processing

### PR DESCRIPTION
**Pull request by hussain-jafari**
### Description
- *Category*: post-processing
- *JIRA issue*: [MIC-3310](https://jira.ihme.washington.edu/browse/MIC-3110)

### Verification and Testing
Ran make_results.

### CODE REVIEW

File: src/vivarium_gates_iv_iron/constants/results.py [hunk 1]
```diff
@@ -84,9 +84,9 @@
 
 POP_STATES = ("living", "dead", "tracked", "untracked")
 SEXES = ("female",)
-YEARS = tuple(range(2024, 2041))
+YEARS = tuple(range(2019, 2033))
```
Comment (albrja) [inline src/vivarium_gates_iv_iron/constants/results.py:87]: Did you mean to change this date range or was this for a test run?

**Review by albrja — APPROVED** [review]

File: src/vivarium_gates_iv_iron/constants/results.py [hunk 2]
```diff
@@ -84,9 +84,9 @@
...
 AGE_GROUPS = (
-    "5_to_9",
+    "0_to_10",
     "10_to_14",
```
Comment (mattkappel) [inline src/vivarium_gates_iv_iron/constants/results.py:90]: A little weird that our age group here overlaps with the next one, isn't it?

File: src/vivarium_gates_iv_iron/constants/results.py [hunk 3]
```diff
@@ -123,6 +123,7 @@
     "MATERNAL_HEMORRHAGE_STATE": models.MATERNAL_HEMORRHAGE_STATES,
     "WITH_MATERNAL_HEMORRHAGE_STATE": models.MATERNAL_HEMORRHAGE_STATES[:-1],
     "ANEMIA_LEVEL": data_values.ANEMIA_DISABILITY_WEIGHTS.keys(),
+    "ANEMIA_LEVEL_STR": models.ANEMIA_LEVELS,
     "SUPPLEMENTATION": models.SUPPLEMENTATION_CATEGORIES,
```
Comment (mattkappel) [inline src/vivarium_gates_iv_iron/constants/results.py]: What does `ANEMIA_LEVEL_STR` have that `ANEMIA_LEVEL` doesn't?

File: src/vivarium_gates_iv_iron/constants/results.py (hunk 1 above)
Comment (hussain-jafari) [reply src/vivarium_gates_iv_iron/constants/results.py:87]: These are the years specific to the stillbirth v&v run.

File: src/vivarium_gates_iv_iron/constants/results.py (hunk 2 above)
Comment (hussain-jafari) [reply src/vivarium_gates_iv_iron/constants/results.py:90]: Yes! Will update.

**Review by rmudambi — APPROVED** [review]

**Pull request merged**